src/components/dashboards/EditModal.jsx

```diff
@@ -66,20 +138,8 @@ const EditModal = props => {
                 <div>
                     Select data
                 </div>
-                <Select style={{ width: 120 }} onChange={updateDataProps} defaultValue={dataId}>
-                    <Select.Option
-                        key={undefined}
-                        value={undefined}>
-                        None
-                    </Select.Option>
-                    {props.dataIds.map(data => {
-                        return (
-                            <Select.Option key={data._id} value={data._id}>
-                                {data.title}
-                            </Select.Option>
-                        )
-                    })}
-                </Select>
+                <DataDropdown onSelectData={updateDataProps} currentData={props.el.dataProps.dataTitle}/>
```

Comment:
right now, not all charts are required to have dataprops so there will be an error here if dataprops is undefined